src/header/Header.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { StatusBar, StyleSheet, Text, View } from 'react-native';
+import isEmpty from 'lodash.isempty';
+import Icon from '../icons/Icon';
+
+export const DummyNavButton = () => (
+  <View style={styles.dummyNavBtn} />
+);
+
+export const Title = (props) => {
+  const {
+    text,
+  ...attributes,
+  } = props;
+
+  return (
+    <Text
```

Comment:
Let's use text from react native elements as it handles font size based on pixel ratio